README.md

```diff
@@ -97,6 +100,9 @@ the body:
 As this is the correct password, the user will be logged in. In the background, his information will be migrated to
 Keycloak.
 
+If the plugin is configured to use the user id as the path parameter for the credential verification request, a `POST`
```

Comment:
I think this paragraph should be moved back right before `As this is the correct password...` for better flow. I would also reword it slightly:
```
If the plugin is configured to use the user id as the path parameter for the
credential verification request, the `POST` request will be performed to
`http://www.old-legacy-system.com/auth/12345678`, instead.
```